barman/cloud_providers/google_cloud_storage.py

```diff
@@ -717,17 +783,6 @@ def identifier(self):
         """
         return self.snapshot_name
 
-    @property
-    def device(self):
```

Comment:
This should also be removed from the base class in [barman/cloud.py](https://github.com/EnterpriseDB/barman/pull/793/commits/ff94501abec1a1f6b849692e44e15ca021ab2f3f#diff-3b02e947f51597b51467015d57554983c2b8ad65fdbc77048ccad8d32bcb5cd2L2419-L2431)